vulnerabilities/importers/gitlab.py

```diff
@@ -71,31 +71,45 @@ def fork_and_get_dir(url):
     return fetch_via_vcs(url).dest_dir
 
 
-class ForkError(Exception):
-    pass
-
-
-class GitLabAPIImporter(Importer):
+class GitLabGitImporter(GitImporter):
```

Comment:
@ziadhany this will change the qualified name for importer and the databases will still have the old qualified name stored in them, which will now not be recogonised by improver, either write a migration to support this change or do not change the name of importer.